Gopkg.toml

```diff
@@ -17,21 +17,21 @@ required = [
   name = "gopkg.in/yaml.v2"
   version = "v2.2.1"
 
-[[constraint]]
+[[override]]
```

Comment:
We need to bump `go-containerregistry` to 1.12.6 for moving those back to `constraint`